docker-compose.override.yml

```diff
@@ -1,12 +1,43 @@
-version: '3'
+version: '3.4'
 
 services:
+  vc-db:
+    image: mcr.microsoft.com/mssql/server:latest
+    ports:
+      - "${DOCKER_SQL_PORT:-1433}:1433"
+    expose:  
+      - 1433  
+    environment: 
+      - ACCEPT_EULA=Y
+      - MSSQL_PID=Express
+      - SA_PASSWORD=v!rto_Labs!
+    volumes:
+      - db-volume:/var/opt/mssql   
+    networks:
+      - virto
+
   virtocommerce.platform.web:
-    environment:
-      - ASPNETCORE_ENVIRONMENT=Development
     ports:
-      - "80"
+      - "${DOCKER_PLATFORM_PORT:-8090}:80"
+    environment:
+     - VirtoCommerce:AllowInsecureHttp=true
+     - ConnectionStrings__VirtoCommerce=Data Source=vc-db;Initial Catalog=VirtoCommerce3docker;Persist Security Info=True;User ID=sa;Password=v!rto_Labs!;MultipleActiveResultSets=False;
```

Comment:
There is SA_PASSWORD variable for the "v!rto_Labs!" password, should it be duplicated here?

Reply:
This is done in order to be able to parameterize the value, as it is defined above in the service for creating mssql

Reply:
Yes, I think we should ensure password in mssql and in connection string should be the same. Otherwise changing password would require changes in 2 places. Could we make it by having one variable and using it in both places?